public/gdb/impostor-syndrome.md

```diff
@@ -60,47 +60,47 @@ The world is *full* of influences put in place to fill us with doubt and keep us
 
 If you feel like you are not a binary woman, than you are not a binary woman. Cis women do not experience that detachment.
 
 ### Toxic Masculinity
 
-Male assigned kids grow up positively drenched in messaging of what it is to "be a man". There are so few examples of positive masculinity in popular media, and AMAB Masculine Enbies are also commonly erased in trans representation that being a gender queer male can feel very lonely. AMAB Enbies are often either grouped in with gay cis men or treated like trans women.
+Male-assigned kids grow up positively drenched in messaging of what it is to "be a man". There are so few examples of positive masculinity in popular media, and AMAB Masculine Enbies are also so commonly erased in trans representation that being a genderqueer male can feel very lonely. AMAB Enbies are often either grouped in with gay cis men or treated like trans women.
 
-You can just be gender-queer! Your identity is valid!
+You can just be genderqueer! Your identity is valid!
 
 ### Transmedicalism
 
 This one hits *everybody*. Transmedicalism (aka Truetrans) is a transgender ideology derived from the Harry Benjamin scale (ranks 5 and 6). It seeks to reinforce the pre-WPATH rules, requiring intense physical dysphoria, demanding medical transition, and often invalidating all non-binary identities. At its core, transmedicalism is a supremacist concept, elevating binary trans people above the needs of any other gender identity, and a push back against the expansion of the transgender identity. They wish for more gatekeeping than we have today, rail against enbies using the transgender label, and would prefer to see fewer people receive treatment for their gender dysphoria.
 
 To put it succinctly, many transmedicalists hate that the newer generation "has it so easy," despite the fact that many of their ranks are part of that generation. This ideology started among disgruntled trans elders, but has since spread to other binary individuals, particularly among young trans men.
 
-If a trans person's first exposure to transness is a transmedicalist, this can severely set back their own self-acceptance and push them even further into the closet. Transmed's are well known to actually tell people "No, you are not trans."
+If a trans person's first exposure to transness is a transmedicalist, this can severely set back their own self-acceptance and push them even further into the closet. Transmeds are well known to actually tell people "No, you are not trans."
 
 **Do not believe these lies.** They are bully tactics explicitly designed to gaslight and dismiss people's pain for self gratification.
 
-### Trans Exclusionary Reactionary Feminism / Gender Critical Movement / Gender Essentialism
```

Comment:
This was not a mistake, it was intentional. TERF ideology of today bears no resemblance to radical feminism of the 70s and 80s. It is entirely reactionary feminism.